shared/graphql/schema.graphql

```diff
@@ -882,9 +888,12 @@ type UserAdminMutations {
 
 union SetSsoConfigResult = SsoConfig | InvalidInput | OperationError
 
+union BucketSetTabulatorTableResult = BucketConfig | InvalidInput | OperationError
+
 type AdminMutations {
   user: UserAdminMutations!
   setSsoConfig(config: String): SetSsoConfigResult
+  bucketSetTabulatorTable(bucketName: String!, tableName: String!, config: String): BucketSetTabulatorTableResult!
```

Comment:
it seems like this will error when trying to delete a table, bc it will return null, and the result type is non-nullable

Reply:
it returns `BucketConfig`
(though I'm not sure that makes much sense)

Reply:
oh i see now. i guess that works